bin/prettier.js

```diff
@@ -50,6 +50,7 @@ if (!filenames.length && !stdin) {
 function getParser() {
   // For backward compatibility. Deprecated in 0.0.10
   if (argv["flow-parser"]) {
+    console.warn("`--flow-parser` is deprecated. Use `--parser flow` instead.");
```

Comment:
There is another place inside of src/options.js

Could you make this this is written on stderr? Otherwise it's going to trigger invalid output without the `--write` option.